sdk/storage/Azure.Storage.Files.DataLake/src/DataLakePathClient.cs

```diff
@@ -2487,21 +2508,31 @@ internal virtual void SetNameFieldsIfNull()
 
                         do
                         {
-                            jsonResponse =
-                                await DataLakeRestClient.Path.SetAccessControlRecursiveAsync(
-                                    clientDiagnostics: ClientDiagnostics,
-                                    pipeline: Pipeline,
-                                    resourceUri: DfsUri,
-                                    mode: mode,
-                                    maxRecords: options.BatchSize,
-                                    version: Version.ToVersionString(),
-                                    acl: accessControlList,
-                                    async: async,
-                                    continuation: continuationToken,
-                                    forceFlag: options.ContinueOnFailure ? true : (bool?)default,
-                                    cancellationToken: cancellationToken)
-                                .ConfigureAwait(false);
-
+                            try
+                            {
+                                jsonResponse =
+                                    await DataLakeRestClient.Path.SetAccessControlRecursiveAsync(
+                                        clientDiagnostics: ClientDiagnostics,
+                                        pipeline: Pipeline,
+                                        resourceUri: DfsUri,
+                                        mode: mode,
+                                        maxRecords: options.BatchSize,
+                                        version: Version.ToVersionString(),
+                                        acl: accessControlList,
+                                        async: async,
+                                        continuation: continuationToken,
+                                        forceFlag: options.ContinueOnFailure ? true : (bool?)default,
```

Comment:
Just double checking, but `ContinueOnFailure` will have the service send back something like a 207 and not a 4XX?

Reply:
ContinueOnFailure will send back 200s (unless its a blobNotFound, authorization error, or some other valid 4XX error)